js/web/test/test-runner.ts

```diff
@@ -574,7 +574,7 @@ export class ProtoOpTestContext {
   private readonly loadedData: Uint8Array;  // model data, inputs, outputs
   session: ort.InferenceSession;
   readonly backendHint: string;
-  constructor(test: Test.OperatorTest) {
+  constructor(test: Test.OperatorTest, private readonly sessionOptions: ort.InferenceSession.SessionOptions = {}) {
```

Comment:
Do we need to declare a member variable to hold sessionOptions parameter value? Or is it implicitly owned by the class object?

Reply:
The class `ProtoOpTestContext` is only used in this test runner context for each operator test. And unlike C++, in JavaScript an object variable is always a reference to a managed object. there is no ownership